_data/sites.json

```diff
@@ -59,9 +59,9 @@
         "name": "Badoo",
         "url": "https://badoo.com/feedback",
         "difficulty": "hard",
-        "notes": "Go to the url provided above and submit a request, picking 'General question' as subject, or send an email to <a href=\"mailto:DPOcorp.badoo.com\">DPOcorp.badoo.com</a>.",
-        "notes_es-ES": "Entra en la dirección indicada arriba y envía una solicitud, selecionando como asunto 'Pregunta general', o envía un email a <a href=\"mailto:DPOcorp.badoo.com\">DPOcorp.badoo.com</a>.",
-        "notes_pl-PL": "Wejdź na powyższy adres i wyślij zapytanie, wybierając jako temat 'Zapytanie ogólne', lub wyślij e-mail na adres <a href=\"mailto:DPOcorp.badoo.com\">DPOcorp.badoo.com</a>."
+        "notes": "Go to the url provided above and submit a request, picking 'General question' as subject, or send an email to [DPOcorp.badoo.com](mailto:DPOcorp.badoo.com).",
```

Comment:
That doesn't seem like an email... The correct email is DPO@corp.badoo.com (i guess the `@` went missing...

Reply:
done, @daviddavo